tests/e2e/normalize.go

```diff
@@ -104,6 +105,9 @@ func normalizeKRMObject(u *unstructured.Unstructured, project testgcp.GCPProject
 	visitor.replacePaths[".spec.softDeletePolicy.effectiveTime"] = "1970-01-01T00:00:00Z"
 	visitor.replacePaths[".status.observedState.softDeletePolicy.effectiveTime"] = "1970-01-01T00:00:00Z"
 
+	// Specific to Compute SSL Certs
```

Comment:
Do we still need this? https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/a5521319b22a945b9d6a5cfc33634121e6a0f09f/tests/e2e/normalize.go#L68

Reply:
No I don't think we need it anymore but let's see 😛